### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by GhoulZoid** [smartcontractkit/full-blockchain-solidity-course-py#701] [Q&A]
![image](https://user-images.githubusercontent.com/95380582/148275169-c81633df-7fde-4bee-8dad-99077ea3a3e9.png)
Any knowledge on this issue?

**Comment by cromewar** [comment]
Hello @GhoulZoid could you share more information about the error please? 
But so far it seems to be a problem with your brownie installation, could you please install it using a virtual env?
